src/coinbase/staking_operation.ts

```diff
@@ -36,14 +37,76 @@ export class StakingOperation {
   }
 
   /**
-   * Get the staking operation ID.
+   * Returns the Staking Operation ID.
    *
-   * @returns The unique ID of the staking operation.
+   * @returns The Staking Operation ID.
    */
   public getID(): string {
     return this.model.id;
   }
 
+  /**
+   * Returns the Wallet ID if it exists.
+   *
+   * @returns The Wallet ID.
+   */
+  public getWalletID(): string | undefined {
+    return this.model.wallet_id;
+  }
+
+  /**
+   * Returns the Address ID.
+   *
+   * @returns The Address ID.
+   */
+  public getAddressID(): string {
+    return this.model.address_id;
+  }
+
+  /**
+   * Returns the Status of the StakingOperation.
+   *
+   * @returns The Status of the StakingOperation.
+   */
+  public getStatus(): StakingOperationStatus | undefined {
+    switch (this.model.status) {
```

Comment:
Is this the best way to do this in typescript?

Reply:
Why not use status directly?

Reply:
Oops, the case statements where suppose to be of type `StakingOperationStatusEnum` which is what comes from the model from the API call. Updating.